### PR TITLE
Emulators, part 1

### PR DESCRIPTION
**Pull request by seal331**
This PR adds 3 emulators and 4 interoperability tools for an old Unix running on one of them.

I'll promise to try to find more mainstream less obscure emulators next time :)

Just as last time, welcome to any changes.

### CODE REVIEW

File: emulators/4-s4-3b1-pc7300.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+patch -Np1 -i ../s4-3b1-pc7300-efe55b-build-module-unconditionally.patch
+```
+
+Then, apply a patch to fix building the kernel module:
+```Bash
+patch -Np1 -i ../s4-3b1-pc7300-efe55b-fix-kernel-module.patch
+```
+
+Apply a patch to build the supporting library as a shared library instead of a static one:
+```
```
Comment (zeckma-tech) [inline emulators/4-s4-3b1-pc7300.md]: Add Bash after backticks.

File: emulators/4-s4-3b1-pc7300.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+patch -Np1 -i ../s4-3b1-pc7300-efe55b-shared-lib.patch
+```
+
+Compile the kernel module:
+```
```
Comment (zeckma-tech) [inline emulators/4-s4-3b1-pc7300.md]: Add Bash after backticks.

File: emulators/4-s4-3b1-pc7300.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+cd sysv &&
+make -f Makefile.daveb running
+```
+
+Now, as the ***root*** user, install the kernel module:
+```
```
Comment (zeckma-tech) [inline emulators/4-s4-3b1-pc7300.md]: Add Bash after backticks.

File: emulators/3-freebee.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+## Installation of freebee
+Optionally, before unpacking, move the distribution zipfiles to a more sensible name:
+```Bash
+mv b727767d2ac48b86f984bf816dfbad9ece94776f.zip freebee-b72776.zip &&
+mv fc7a6fc602e2fbcd24851670a5242358765feacf.zip musashi-fc7a6f.zip
+```
+
+Unpack the musashi submodule into the right place:
+```
```
Comment (zeckma-tech) [inline emulators/3-freebee.md]: Add Bash after backticks.

**Review by Zeckmathederg — CHANGES_REQUESTED** [review]

File: emulators/3-freebee.md (hunk 4 above)
Comment (seal331) [reply emulators/3-freebee.md]: resolved in commit a45ea43

File: emulators/4-s4-3b1-pc7300.md (hunk 2 above)
Comment (seal331) [reply emulators/4-s4-3b1-pc7300.md]: resolved in commit a45ea43

File: emulators/4-s4-3b1-pc7300.md (hunk 3 above)
Comment (seal331) [reply emulators/4-s4-3b1-pc7300.md]: resolved in commit a45ea43

File: emulators/4-s4-3b1-pc7300.md (hunk 1 above)
Comment (seal331) [reply emulators/4-s4-3b1-pc7300.md]: resolved in commit a45ea43

**Review by Zeckmathederg — APPROVED** [review]
LGTM. Merging. Thanks!

**Pull request merged**